week2/homework/src/index.js

```diff
@@ -1,3 +1,26 @@
+#!/usr/bin/env node
 'use strict';
 
-// TODO: Write the homework code in this file
+const args = require('./arguments');
+const method = require('./methods');
+
+let param = args.file[1] + '\r\n';
+
+const command = args.file[0];
+
+switch (command) {
+  case 'add':
+    method.add(param);
+    break;
+  case 'list':
+    method.list();
+    break;
+  case 'remove':
+    method.remove(param);
+    break;
+  case 'reset':
+    method.reset();
+    break;
+  default:
```

Comment:
When you run `node index.js` or `node index.js` the output looks like the following:
```
{ add: [Function: add],
  list: [Function: list],
  remove: [Function: remove],
  reset: [Function: reset] }
```
Can you change it so that it will display some instructions of usage for each commands?